GOVERNANCE.md

```diff
@@ -0,0 +1,104 @@
+# Process for becoming a maintainer
+

```

Comment:
Add a line like "Depending on the status of your current organization, the process for becoming a maintainer may differ as described below: "